components/dashboard/src/projects/Project.tsx

```diff
@@ -171,10 +171,10 @@ export default function () {
                     return <Item key={`branch-${index}-${branchName}`} className="grid grid-cols-3 group">
                         <ItemField className="flex items-center">
                             <div>
-                                <div className="text-base text-gray-900 dark:text-gray-50 font-medium mb-1">
+                                <a href={branch.url}><div className="text-base text-gray-900 dark:text-gray-50 font-medium mb-1">
```

Comment:
**issue**: This is missing a hover state! 🙀

We could go with something subtle for now like the following. What do you think?

```suggestion
                                <a href={branch.url}><div className="text-base text-gray-600 hover:text-gray-800 dark:text-gray-50 dark:hover:text-gray-200 font-medium mb-1">
```